src/tsolvers/lasolver/Tableau.cc

```diff
@@ -13,11 +15,10 @@
 
 using namespace opensmt;
 namespace {
-    template<class C, class E>
-    inline bool contains(const C & container, const E & elem){
-        return container.find(elem) != container.end();
-    }
+template <class C, class E> inline bool contains(const C & container, const E & elem) {
+    return container.find(elem) != container.end();
 }
+} // namespace
```

Comment:
What is the point of this comment btw?

Reply:
Huh, I have not noticed it before. It has been probably added by clang-format, but I will double-check it.

Reply:
Looks like this is the the style people at LLVM are converging to: https://llvm.org/docs/CodingStandards.html#anonymous-namespaces

There have been some discussions, but now they seem to comment the closing brace of namespace X with
`// namespace X`
Since this is anonymous namespace, there is no name at the end.

In clang-format, this is the setting `FixNamespaceComments`

I generally agree that it is good to annotate the end of namespace.
Maybe we can rise the limit for this, which is specified by setting `ShortNamespaceLines`.

Reply:
Hmm, ok, that's a very interesting point